renku/core/models/workflow/plan.py

```diff
@@ -192,11 +194,13 @@ def convert_output(output: CommandOutputTemplate) -> CommandOutput:
 
             return CommandOutput(
                 id=output._id.replace(self.id_, run_id),
-                produces=entity,
+                create_folder=output.create_folder,
```

Comment:
Do we intend to merge my default_value PR before this or to merge this one instead of the default_value PR?

In the latter case, I think we should set default_value here instead of produces and remove produces.
What do you think?

Reply:
That PR is already merged. This is `CommandOutput` and I believe we did not remove `produces` field there since it belongs to the old metadata.

Reply:
Right.. For some reason in my head that was already gone. Wishful thinking :sweat_smile: